packages/perseus/src/widgets/categorizer/categorizer.tsx

```diff
@@ -328,4 +329,7 @@ export default {
     // TODO(LEMS-2656): remove TS suppression
     // @ts-expect-error: Type 'UserInput' is not assignable to type 'PerseusCSProgramUserInput'.
     scorer: scoreCategorizer,
+    // TODO(LEMS-2656): remove TS suppression
+    // @ts-expect-error: Type 'UserInput' is not assignable to type 'PerseusCSProgramUserInput'.
```

Comment:
This is the same issue as with the `scorer` entry. Each of our validation functions accepts the specific user input and validation data type for the widget, but our function definition for it on `WidgetExports` takes a type union of _any_ widget's user input and validation data type. 

The solution, I think, is to introduce more generic types on WidgetExports<T> but that gets out of hand quickly each time I try.  I'm going to leave this here for now and come back to it after this PR.

Reply:
Might be a fun "wind down" task next week. Might not. idk, I'm fine with error suppression for now.